alf/algorithms/rl_algorithm.py

```diff
@@ -452,8 +452,10 @@ def unroll(self, unroll_length):
             # to store it in replay buffers
             transformed_time_step = transformed_time_step._replace(
                 untransformed=time_step)
+            self.rollout()
```

Comment:
We should avoid adding overhead like this. Recursive call can be expensive. Perhaps modify common.mark_training to handle all three modes.